week-1-ts-ecosystem/w01e01-typescript-basics/src/utils/index.ts

```diff
@@ -0,0 +1,11 @@
+import { User } from "../types";
+
+export function hasAddress(user: User): boolean {
+  return Boolean(user.address);
+}
+
+function hasGivenAge(requiredAge: number): (user: User) => boolean {
+  return (user: User): boolean => user.age >= requiredAge;
+}
+
+export const isAdult: (user: User) => boolean = hasGivenAge(18);
```

Comment:
Znów dość ogólna nazwa `utils`.

Generalnie Twój podział per pliki - jest dobry.

Poprawiłbym jedynie ich nazwy - żeby "Więcej zdradzały".

Tutaj innym sugerowałem takie minimum:
`user.utils.ts`
albo jeszcze lepiej, zauważyć, że to `validators`
więc:
`user.validators.ts`

Reply:
To wiadomo, taka sama sytuacja jak wyżej 👍

Reply:
Zmieniłem nazwy plików 👍